### DISCUSSION TITLE
Documentation unclear

### DISCUSSION THREAD

**Discussion by boonstoppel** [clickbar/laravel-magellan#44] [Ideas]
In the documentation I can see the functions Point::make and ST::distanceSphere but you don't explain where these functions are defined. 

I figured out that Point is in Clickbar\Magellan\Data\Geometries\Point:

```
use Clickbar\Magellan\Data\Geometries\Point;
Point::makeGeodetic(50.107471773560114, 8.679861151457937);
```

However it's unclear to me where ST for distanceSphere is defined:

```
use Clickbar\Magellan\xxx\xxx\??;
$query->stSelect(ST::distanceSphere($currentShipPosition, 'location'), 'distance_to_ship');
```

**Comment by saibotk** [comment]
Oh, sorry we will try to update the README to make that more clear in the future then.

The ST class is defined here:
https://github.com/clickbar/laravel-magellan/blob/df480a3ece785a4976610a166d95ac23938ea38e/src/Database/PostgisFunctions/ST.php

it can be imported like so:
```php
use Clickbar\Magellan\Database\PostgisFunctions\ST;

$query->stSelect(ST::distanceSphere($currentShipPosition, 'location'), 'distance_to_ship');
```

i hope this helps :) and thanks for checking out magellan 👍

**Reply by boonstoppel** [reply]
Thank you!
